tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"compilerOptions": {
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"importHelpers": true,
+		"module": "esnext",
+		"moduleResolution": "node",
+		"removeComments": true,
+		"resolveJsonModule": true,
+		"strict": true,
+		"target": "es2018"
+	},
+	"include": [
+		"event-handler/**/*",
+		"middleware/**/*",
+		"scripts/**/*",
+		"sign/**/*",
+		"verify/**/*"
+	]
```

Comment:
Should it be considered to move all the project to `src/*` so we can do this?
```json
   "include": ["src/**/*"]
```

Reply:
That would greatly simplify things

Reply:
Yes, I did the same with all the other repos as part of the migration to TypeScript